src/components/Todo.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import './Todo.css';
+
+const Todo = props => {
```

Comment:
I dig that you converted this to a functional component and moved state to the parent component! Well done.